packages/synapse-react-client/src/components/download_list/TableQueryDownloadConfirmation.test.tsx

```diff
@@ -5,45 +5,30 @@ import * as ToastMessage from '../ToastMessage/ToastMessage'
 import {
```

Comment:
This test suite was the primary motivation for this change; the `TableQueryDownloadConfirmation` issues multiple different queries on the same table. I couldn't mock this behavior in MSW without the other changes in this PR.